OpenRA.Mods.Common/Activities/ExternalCaptureActor.cs

```diff
@@ -19,21 +19,28 @@ namespace OpenRA.Mods.Common.Activities
 	class ExternalCaptureActor : Activity
 	{
 		readonly ExternalCapturable capturable;
+		readonly Actor underlyingActor;
```

Comment:
Please take the actor/frozen actor directly from the `target` instead of caching it here.